src/linagora.esn.unifiedinbox/app/services/close-window/close-window.service.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const RETURN_VALUE = 'Are you sure you want to leave?';
```

Comment:
i18n it. Also explain what will happen if use closes the window

Reply:
The close window service will test if the composer is opening or now via getting the status of inboxComposerStatus 
->If it is OPENING the message will be fired 
->If not (DISCARDING)  will exit from function and the message will not be fired